spec/factories/task.rb

```diff
@@ -132,6 +130,33 @@
         }
       end
     end
+    trait(:with_meta_data) do
+      meta_data do
+        {
+          'meta-data' => {
+            '@xmlns' => {'namespace' => 'custom_namespace.org'},
+            'namespace:meta' => {
+              '$1' => 'data',
+               '@xmlns' => {'namespace' => 'custom_namespace.org'},
```

Comment:
For me, it seems like this line (and some others in this trait) have the wrong indention.